docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
```

Comment:
It should probably be `/plugins/create?name=foo`, as in with `name` as a GET parameter. To follow the reasoning in (1) above, that's performing an action against the resource `/plugins/create`, but with `name` as an additional option.

It would typically be in the request body, but in this case that's impossible.

(I would also argue this should be just `POST /plugins`, but I'm happy to fix this in one big go in #25015. ;)